### ISSUE TITLE
[Background Media]: QA: Focus is not visible on 'Play' button

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8995]
**Engineering info:**  
- Innovation Team engineer: @andy-blum 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2490
- DPO consulting engineer: N/A

-----

### Description

Page Element:
Play button

Issue Description:
Keyboard focus is not visible clearly on the Play button

Expected:
Keyboard focus should be visible clearly on the button.

User Impact:
When focus indicator for the page element is not clear, the user would have difficulty in identifying the currently focused element. this impacts Keyboard only users and people who with low vision

### Component(s) impacted

Background Media

### Browser

Firefox

### Carbon for IBM.com version

v1.35.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-react

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components-react

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by RichKummer** [comment]
The `Background media` design specs do not have a focus state to work off of. Putting together a design exploration story to solve for the contrast issue on the focus state.

**Comment by RichKummer** [comment]
@andy-blum @proeung - I've updated the `Background media` [**design specs**](https://ibm.box.com/s/t2y4gne6754xuf3popy9vfhr9rhupm0v) to define the focus state for the White theme. In the PDF, the relevant page is 13 and the info is also screencapped here:

<img width="984" alt="Screen Shot 2022-06-23 at 3 35 17 PM" src="https://user-images.githubusercontent.com/15643582/175384409-9034f50e-716d-4138-b896-6daa7d773b0a.png">

If you have any questions, please let me know!

**Comment by proeung** [comment]
@RichKummer Thanks for updating the specs! We'll go ahead and move this issue into this sprint, now that we have the proper specs from design.

**Comment by gopinathan-n** [comment]
@IgnacioBecerra Double check change log.

**Comment by IgnacioBecerra** [comment]
@Gopi916 Staging storybook was slightly outdated -- the fix should be showing on staging now.
